Run handlers before configuring veth interfaces

Alex-Welsh opened the pull request:
Handlers from MichaelRigart.interfaces are not run until the end of the play, which results in the veth role trying to create veth interfaces against a non-existing bridge.

Change-Id: Ie1c9e1ada50774cbcaec528bd64d8df066c21d2e

markgoddard commented on ansible/roles/kolla-openstack/defaults/main.yml:
```diff
@@ -15,6 +15,15 @@ kolla_enable_glance:
+# Grafana configuration.
+
+# Whether to enable Grafana.
+kolla_enable_grafana:
```
There needs to be a default for this here: https://github.com/stackhpc/kayobe/blob/master/ansible/group_vars/all/kolla#L272. Might be best to check what the default is for enable_grafana in kolla-ansible.

dougszumski replied:
Thanks - ready for re-review.

priteau committed:
Run handlers before configuring veth interfaces

Handlers from MichaelRigart.interfaces are not run until the end of the
play, which results in the veth role trying to create veth interfaces
against a non-existing bridge.

Change-Id: Ie1c9e1ada50774cbcaec528bd64d8df066c21d2e

bbezak commented:
superseded by upstream sync https://github.com/stackhpc/kayobe/pull/110
